simplified-app-simplye/build.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
+apply from: '../ktlint.gradle'
```

Comment:
Is this actually the correct module to which linting should be applied?

I would expect the bulk of linting to be applied in all of the other modules; `simplified-app-simplye` contains very little actual source code.